pysteps/tests/test_extrapolation_semilagrangian.py

```diff
@@ -19,3 +19,13 @@ def test_semilagrangian():
     # result
     result = extrapolate(precip, velocity, num_timesteps)
     assert_array_almost_equal(result, expected)
+
+def test_ascending_time_step():
+    precip = np.ones((8, 8))
+    v = np.ones((8, 8))
+    velocity = np.stack([v, v])
+
+    not_ascending_timesteps = [1,2,3,5,4,6,7]
+    with pytest.raises(ValueError):
```

Comment:
careful: the module `pytest` was not imported at the top

Reply:
@pierrejavelle you only need to `import pytest` at top of this file and we'll be able to merge your pull request, please let me know if something is not quite clear.